torch/csrc/distributed/autograd/init.cpp

```diff
@@ -75,14 +75,18 @@ PyObject* dist_autograd_init(PyObject* /* unused */) {
   module.def(
       "_release_context",
       [](int64_t context_id) {
-        return DistAutogradContainer::getInstance().releaseContext(context_id);
+        return DistAutogradContainer::getInstance().releaseContextIfPresent(context_id);
       },
       py::call_guard<py::gil_scoped_release>());
 
   module.def("_get_max_id", []() {
     return DistAutogradContainer::getInstance().getMaxId();
   });
 
+  module.def(
+      "_setContextCleanupTimeout",
+      [](int64_t newTimeout) { DistAutogradContainer::getInstance().setCleanupContextTimeout(newTimeout); });
```

Comment:
We usually do `std::chrono::duration` (i.e. `std::chrono::seconds` and the like) when taking in timeout arguments. And then from python a timedelta is passed it, and pybind11 will handle timedelta --> std::chrono conversion if yoy include `pybind11/chrono.h` (https://pybind11.readthedocs.io/en/stable/advanced/cast/chrono.html)